src/content/platform-integration/web/bootstrapping.md

```diff
@@ -0,0 +1,244 @@
+can use this script by simply placing an async script tag for it in your
+`index.html` file in the `web` subdirectory of your flutter project:
+
+```html
+<html>
+  <body>
+    <script src="flutter_bootstrap.js" async></script>
+  </body>
+<html>
+```
+
+Alternatively, you can inline the entire contents of the `flutter_bootstrap.js`
+file by inserting the template token `{% raw %}{{flutter_bootstrap_js}}{% endraw %}` in your
+`index.html` file:
+
+```html
+<html>
+  <body>
+    <script>
+      {% raw %}{{flutter_bootstrap_js}}{% endraw %}
+    </script>
+  </body>
+<html>
+```
+
+The `{% raw %}{{flutter_bootstrap_js}}{% endraw %}` token will be replaced with the contents of the
+`flutter_bootstrap.js` file when the `index.html` file is copied to the output
+directory during the build step.
+
+## Customizing Initialization
+
+By default, the `flutter build web` process generates a `flutter_bootstrap.js`
+file that does a simple initialization of your Flutter app. However, in some
+scenarios, you may have a reason to customize this initialization process, such
+as:
+
+* Setting a custom Flutter configuration for your app
+* Changing the settings for the Flutter service worker
+* Write custom JavaScript code to run at different stages of the startup process
+
+To write your own custom bootstrapping logic instead of using the default script
+produced by the build step, you can place a `flutter_bootstrap.js` file in the
+`web` subdirectory of your project, which will be copied over and used in lieu
+of the default script produced by the build. This file is also templated, and
+you can insert several special tokens which the build step will substitute at
+build time when copying the `flutter_bootstrap.js` file to the output directory.
+Here is a list of the tokens that the build step will substitute in either the
+`flutter_bootstrap.js` or `index.html` files:
+
+<div class="table-wrapper" markdown="1">
+
+| Token | Replaced With |
+|-|-|
+| `{% raw %}{{flutter_js}}{% endraw %}` | The JavaScript code that makes the `FlutterLoader` object available in the `_flutter.loader` global variable. (See the `_flutter.loader.load() API` section below for more details.) |
+| `{% raw %}{{flutter_build_config}}{% endraw %}` | A JavaScript statement that sets metadata produced by the build process which gives the `FlutterLoader` information needed to properly bootstrap your application. |
+| `{% raw %}{{flutter_service_worker_version}}{% endraw %}` | A unique number representing the build version of the service worker, which can be passed as part of the service worker configuration (see the "Service Worker Settings" table below). |
+| `{% raw %}{{flutter_bootstrap_js}}{% endraw %}` | As mentioned above, this inlines the contents of the `flutter_bootstrap.js` file directly into the `index.html` file. Note that this token can only be used in the `index.html` and not the `flutter_bootstrap.js` file itself. |
+{:.table}
```

Comment:
All of the columns in these tables are...weird.

The left column is too big so the text on the right is too small.

🤷

Reply:
Fixed!!